src/ToolingSharedLibrary/CurrentCodeGenerationState/Enclave/Untrusted.h

```diff
@@ -20,14 +20,7 @@ namespace CodeGenTest
             in_flatbufferT.m_arg5 = VbsEnclaveABI::Shared::Converters::ConvertType<decltype(in_flatbufferT.m_arg5)>(arg5);
             in_flatbufferT.m_arg7 = VbsEnclaveABI::Shared::Converters::ConvertType<decltype(in_flatbufferT.m_arg7)>(arg7);
             in_flatbufferT.m_arg9 = VbsEnclaveABI::Shared::Converters::ConvertType<decltype(in_flatbufferT.m_arg9)>(arg9);
-
-            using ParamsT = decltype(in_flatbufferT);
-            auto flatbuffer_builder = VbsEnclaveABI::Shared::PackFlatbuffer(in_flatbufferT);
-            using ReturnParamsT = FlatbuffersDevTypes::FuncWithAllArgs_1_argsT;
-            auto function_result = ReturnParamsT();
-            THROW_IF_FAILED((VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<ParamsT, ReturnParamsT>("CodeGenTest::AbiDefinitions::FuncWithAllArgs_1_Generated_Stub", flatbuffer_builder, function_result)));
-            
-            auto return_params = VbsEnclaveABI::Shared::Converters::ConvertStruct<FuncWithAllArgs_1_args>(function_result);
+            auto return_params = VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<FuncWithAllArgs_1_args>(in_flatbufferT, "CodeGenTest::AbiDefinitions::FuncWithAllArgs_1_Generated_Stub");
```

Comment:
>auto return_params = VbsEnclaveABI::Enclave::CallVtl0CallbackFromVtl1<FuncWithAllArgs_1_args>(in_flatbufferT, "CodeGenTest::AbiDefinitions::FuncWithAllArgs_1_Generated_Stub"); [](http://example.com/codeflow?start=12&length=175)

excellent yes.  this feels much more intuitive that CallVtl0CallbackFromVtl1 operates only in terms of DevTypes and hides all the PackFlatbuffer/on-the-wire logic.